### DISCUSSION TITLE
'limit' params

### DISCUSSION THREAD

**Discussion by iveinsomnia** [br-g/openf1#27] [Ideas]
is there a way to "limit" data recieved to spare time and bandswitch? I'm mostly thinking about weather for exemple.

**Comment by br-g** [comment]
Hello,
Thanks for trying to save resources!
I recommend that you filter on date to fetch only the latest data.

For example, use `https://api.openf1.org/v1/weather?session_key=latest&date>{date}` to fetch the latest weather data, with `{date}` the date of the latest data you previously fetched.
